client/src/pages/teds/charts/PrctIpccPublicationsForFiveCountries.tsx

```diff
@@ -0,0 +1,219 @@
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Sciences Physique et Adaptations",
+  };
+
+  const filter_wg2_and_wg2cross = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations et Adaptations - risques locaux",
+  };
+  const filter_wg2_and_wg3 = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+    },
+    name: "Adaptation et Atténuation",
+  };
+
+  const filters = [
+    filter_wg1,
+    filter_wg2,
+    filter_wg2cross,
+    filter_wg3,
+    filter_wg1_and_wg2,
+    filter_wg2_and_wg2cross,
+    filter_wg2_and_wg3,
+  ];
+
+  const countries = [
+    "United States",
+    "United Kingdom",
+    "Germany",
+    "France",
+    "China",
+  ];
+
+  const responses = [
+    useQueryResponse(filters[0].body, 50, filters[0].name),
+    useQueryResponse(filters[1].body, 50, filters[1].name),
+    useQueryResponse(filters[2].body, 50, filters[2].name),
+    useQueryResponse(filters[3].body, 50, filters[3].name),
+    useQueryResponse(filters[4].body, 50, filters[4].name),
+    useQueryResponse(filters[5].body, 50, filters[5].name),
+    useQueryResponse(filters[6].body, 50, filters[6].name),
+  ];
+
+  const response_total = useQueryResponse(
+    filter_ipcc.body,
+    50,
+    filter_ipcc.name
+  );
+
+  const loadings = responses.map((response) => response.isLoading);
+  const isLoad = (currentValue) => currentValue === true;
+
+  if (loadings.some(isLoad)) {
+    return <div>Loading...</div>;
+  }
+
+  const buckets = responses.map((response) =>
+    response.data.aggregations.by_countries.buckets.filter((country) =>
+      countries.includes(country.key)
+    )
+  );
+
+  const values = [];
+
+  filters.map((filter, index) => {
+    values.push({
+      name: filter.name,
+      data: [
+        buckets[index].find((country) => country.key === countries[0])
+          .doc_count,
+        buckets[index].find((country) => country.key === countries[1])
+          .doc_count,
+        buckets[index].find((country) => country.key === countries[2])
+          .doc_count,
+        buckets[index].find((country) => country.key === countries[3])
+          .doc_count,
+        buckets[index].find((country) => country.key === countries[4])
+          .doc_count,
+      ],
+    });
+  });
+
+  if (response_total.isLoading) {
+    return <div>Loading...</div>;
```

Comment:
You should have only one `return <div>Loading...</div>`. Cumulate the conditions in the if.